servers/cromwell/jobs/util.py

```diff
@@ -1,8 +1,11 @@
-from typing import GenericMeta
-from datetime import datetime, date
+from datetime import date, datetime
+
 from six import integer_types, iteritems
 
 
+class GenericMeta(type):
```

Comment:
I found this as a suggested workaround for the removal of GenericMeta in python 3.8 [here:](https://github.com/anchore/anchore-engine/issues/611)